ltc/logs/command_factory/logs_command_factory_test.go

```diff
@@ -60,6 +65,17 @@ var _ = Describe("CommandFactory", func() {
 			Expect(outputBuffer).To(test_helpers.Say("Incorrect Usage"))
 			Expect(fakeTailedLogsOutputter.OutputTailedLogsCallCount()).To(Equal(0))
 		})
+
+		It("handles non existent appguids", func() {
```

Comment:
the testcase will be output in cmdline. so i think the testcase name should be handles non existent app instead of appguids.